src/BinSkim.Rules/ELFRules/BA3005.EnableStackClashProtection.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+
+using Microsoft.CodeAnalysis.BinaryParsers;
+using Microsoft.CodeAnalysis.IL.Sdk;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.IL.Rules
+{
+    [Export(typeof(Skimmer<BinaryAnalyzerContext>)), Export(typeof(ReportingDescriptor))]
+    public class EnableStackClashProtection : ELFBinarySkimmerBase
+    {
+        /// <summary>
+        /// BA3005
+        /// </summary>
+        public override string Id => RuleIds.EnableStackClashProtection;
+
+        /// <summary>
+        /// This check ensures that stack clash protection is enabled. 
+        /// Each program running on a computer uses a special memory region called the stack. 
```

Comment:
>the stack [](http://example.com/codeflow?start=83&length=9)

align all the text with this. check other files as example

Reply:
fixed